tests/foreman/cli/test_remoteexecution.py

```diff
@@ -344,6 +347,89 @@ def test_positive_run_recurring_job_with_max_iterations_by_ip(self, rex_contenth
         assert rec_logic['state'] == 'finished'
         assert rec_logic['iteration'] == '2'
 
+    @pytest.mark.tier3
+    @pytest.mark.rhel_ver_list([8])
```

Comment:
```suggestion
    @pytest.mark.rhel_ver_list('8')
```

Reply:
@shweta83 Has there been any change regarding this fixture? Because I can see many instances of the [8] form around robottelo